server/src/test/java/org/elasticsearch/action/ingest/SimulatePipelineRequestParsingTests.java

```diff
@@ -269,99 +268,4 @@ public void testNotValidDocs() {
         );
         assertThat(e3.getMessage(), containsString("required property is missing"));
     }
-
-    public void testIngestPipelineWithDocumentsWithType() throws Exception {
```

Comment:
We'll still need this test as (unfortunately) this is still available in 9.0 but deprecated.

Reply:
Roger.  Given that this test can no longer reference V_7, I'll update it to `V_8` and then adjust the logic  in [server/src/main/java/org/elasticsearch/action/ingest/SimulatePipelineRequest.java](https://github.com/elastic/elasticsearch/pull/118103/files/e003c4a0090fdb94a75e4f52b196b4a3297077eb#diff-2ee62f54d3610bfbcec37413b8946e2a6d7dbc238e6f735afacad1341d4c5523) as described in [this comment](https://github.com/elastic/elasticsearch/pull/118103#discussion_r1878663095). WDYT?